README.md

```diff
@@ -104,15 +68,13 @@ github "auth0/Auth0.swift" ~> 2.3
 
 Then, run `carthage bootstrap --use-xcframeworks`.
 
-> 💡 For further reference on Carthage, check their [official documentation](https://github.com/Carthage/Carthage#getting-started).
-
-## Getting Started
+### Configure the SDK
 
-### Configuration
+Head to the [Auth0 Dashboard](https://manage.auth0.com/#/applications/) and create a new **Native** application.
```

Comment:
As this is related to setup on manage.auth0.com maybe it could be folded under `Configure Auth0` and then the next pieces are `Configure the SDK`?

Reply:
There's no "Configure Auth0" in this README. Not sure if it's worth having one with a one-liner?